pokerpoll/pokerpoll.py

```diff
@@ -1,56 +1,84 @@
+                    "BritneySpears14: Oh, I like to play dress up."
+                    "bloodninja: Me too baby."
+                    "BritneySpears14: I kiss you softly on your chest."
+                    "bloodninja: I cast Lvl. 3 Eroticism. You turn into a real beautiful woman."
+                    "BritneySpears14: Hey…"
+                    "bloodninja: I meditate to regain my mana, before casting Lvl. 8 Cock of the Infinite."
+                    "BritneySpears14: Funny I still don't see it."
+                    "bloodninja: I spend my mana reserves to cast Mighty F*ck of the Beyondness."
+                    "BritneySpears14: You are the worst cyber partner ever. This is ridiculous."
+                    "bloodninja: Don't f*ck with me bitch, I'm the mightiest sorcerer of the lands."
+                    "bloodninja: I steal yo soul and cast Lightning Lvl. 1,000,000 Your body explodes into a fine bloody mist, because you are only a Lvl. 2 Druid."
+                    "BritneySpears14: Don't ever message me again you piece of ****."
+                    "bloodninja: Robots are trying to drill my brain but my lightning shield inflicts DOA attack, leaving the robots as flaming piles of metal."
+                    "bloodninja: King Arthur congratulates me for destroying Dr. Robotnik's evil army of Robot Socialist Republics. The cold war ends. Reagan steals my accomplishments and makes like it was cause of him."
+                    "bloodninja: You still there baby? I think it's getting hard now."
+                    "bloodninja: Baby?")
+            return None
+
+        if playtime == "now":
+            playhour = dt.now().hour + 1
+            playminute = dt.now().minute
+
+        try:
+            playtime = playtime.lower()
+            twelve_hour = [x for x in ["am", "pm"] if x in playtime]
+
+            if twelve_hour and not len(twelve_hour) > 1:
+                twelve_hour = "".join(twelve_hour)
+                playtime = playtime.replace(twelve_hour, "")
 
-        #Your code will go here
-        poll = await self.bot.say("@here poker time?\n\ni play to win kid\n--------------------------\n\npoker's for nerds\n--------------------------")
+            if ":" in playtime:
+                playhour, playminute = map(int, playtime.split(":"))
+            else:
+                playhour = int(playtime)
+                playminute = 0
+
+            if twelve_hour == "pm":
+                    playhour += 12
+        except ValueError:
+            self.bot.say("wot fukin time is that diked?")
+            return None
+
+        #  Your code will go here
+        poll = await self.bot.say("@here poker time?\n\n"
+            f"i play to win kid {self.dash}"
+            f"poker's for nerds {self.dash}")
 
-        for reaction in reactions:
+        for reaction in self.reactions:
             await self.bot.add_reaction(poll, reaction)
 
         react_hist = {}
+        time_to_duel = False
```

Comment:
perhaps this should be an instance variable? how does cog concurrency work between different servers?

Reply:
the expiry of the poll should be specific to the poll, there could be multiple running for different times.